### ISSUE TITLE
now dev requires login and is missing --token from help

### ISSUE THREAD

**Issue by lucacasonato** [vercel/vercel#3772]
`now dev` requires login now. If you don't log in it gives you a fun error like this:
```
Error! An unexpected error occurred!
  Error: The specified token is not valid
    at Object.getUser (/home/user/.config/yarn/global/node_modules/now/dist/index.js:2:2248948)
    at processTicksAndRejections (internal/process/task_queues.js:94:5) Error: The specified token is not valid
    at Object.getUser (/home/user/.config/yarn/global/node_modules/now/dist/index.js:2:2248948)
    at processTicksAndRejections (internal/process/task_queues.js:94:5)
```

In CI you can specify a token using `-t` or `--token`, but this is not listed in the description shown by specifying `--help`.

**Comment by styfle** [comment]
Thanks for reporting! This will be fixed in PR #3907.

**Comment by styfle** [comment]
You can try this out on the canary channel `npm i -g now@canary`.
